packages/js/src/Modules/Verto/webrtc/helpers.ts

```diff
@@ -33,9 +33,13 @@ const _constraintsByKind = (
  * Retrieve device list using the browser APIs
  * If 'deviceId' or 'label' are missing it means we are on Safari (macOS or iOS)
  * so we must request permissions to the user and then refresh the device list.
+ *
+ * @NOTE Firefox has yet to fully implement audio output devices. As of v63,
```

Comment:
I think you can just write `Browser Compatibility Note:` just like you did in `packages/js/src/Modules/Verto/BrowserSession.ts`